cheat_sheets/aedt_cheat_sheet/pyaedt_cheat_sheet.tex

```diff
@@ -1,22 +1,121 @@
-\documentclass[9pt,landscape]{article}
-\usepackage{{./../static/style}}
```

Comment:
Most of the style packages and renew commands(almost all) has been added in another style file `\usepackage{{./../static/style}}` so tis command will replace all the lines from 1 - 22 & 30-107. also the images are added there.